From ea6bb37b6e281586244b894dd16bae980475b278 Mon Sep 17 00:00:00 2001
From: Sirius902 <10891979+Sirius902@users.noreply.github.com>
Date: Thu, 30 Jan 2025 17:07:25 -0800
Subject: [PATCH] Hardcode output of sw_vers

---
 build.rs | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build.rs b/build.rs
index 3d05d43..fed5be8 100644
--- a/build.rs
+++ b/build.rs
@@ -24,11 +24,7 @@ fn get_macos_major_version() -> Option<usize> {
         return None;
     }
 
-    let output = std::process::Command::new("sw_vers")
-        .args(["-productVersion"])
-        .output()
-        .unwrap();
-    let version = std::str::from_utf8(&output.stdout).unwrap().trim_end();
+    let version = "11.0";
     let components: Vec<&str> = version.split('.').collect();
     let major: usize = components[0].parse().unwrap();
     Some(major)
-- 
2.48.1

